src/main/java/com/hms_networks/americas/sc/extensions/taginfo/TagInfoManager.java

```diff
@@ -371,6 +371,16 @@ private static synchronized void processTagListEBDLine(String line)
         case indexUnit:
           String tagUnit = currentToken;
 
+          // Remove wrapping quotes if present
+          final char quoteChar = '"';
```

Comment:
should this be named doubleQuoteChar to avoid ambiguity?

Reply:
Updated, although I think it is/was fine either way. 

At some point, code needs to be considered self-explanatory, otherwise, it introduces unnecessary redundancy. In situations like this, I don't have a hard preference against including variables such as `doubleQuoteChar`, but I don't think it is better than hardcoded `"`. 

Given the code comment, a hardcoded `"` should have its meaning easily interpreted, and adding variable(s) can arguably make the code more difficult to follow in some situations simply because it results in longer code blocks.